### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by Descacharrado** [knyghtmare/AOI_Rework#11]
- [ ] Proofreading needed #10 
- [ ] On SP, make both player controlled sides be only one.

**Comment by knyghtmare** [comment]
@Descacharrado If you're curious, the `[chat][/chat]` was intentional since you may have observers in MP game who are just watching (as a movie) 
> if it is possible, removing the dialog when observing
would increase the game flow so much more ... Ovservers have to click dozen of times just to watch a game.

chat is more convenient for observers than a message pop-up. 

> Difficulty may be too high, have not played through it all so far tho, so not a task yet.

Cannot really comment on Challenging since it may have been set too high to start with. (since AOI's main complaint was being too easy). Normal is doable. 

> On SP, make both player controlled sides be only one.

Like LOW? Convenient would have to juggle some defines and maybe re-define Sunllis for SP and MP.

> Proofreading needed

16074 words. Good luck!

**Comment by Descacharrado** [comment]
> @Descacharrado If you're curious, the `[chat][/chat]` was intentional since you may have observers in MP game who are just watching (as a movie)

Ah, I see, then only for SP.
> Like LOW? Convenient would have to juggle some defines and maybe re-define Sunllis for SP and MP.

Like LOW but better executed, also it seems to be easier since there SEEM to be only two sides instead of 2, 3 and 4 depending on the map, havent played through it all tho.
> 16074 words. Good luck!

Thank you!

About the SP/MP stuff if you wait long enough I may have a go at it myself.

**Comment by knyghtmare** [comment]
> Like LOW but better executed, also it seems to be easier since there SEEM to be only two sides instead of 2, 3 and 4 depending on the map, havent played through it all tho.

Yeah, keeping the number of sides constant is paramount as it's disappointing when playing as 4 friends and making 1 or 2 of them sit out some scenarios just cause the design wouldn't allow it.

**Comment by knyghtmare** [comment]
Okay, I have added in another of your Feature Requests. Should be more "aesthetic" now.

**Comment by Descacharrado** [comment]
> Okay, I have added in another of your Feature Requests. Should be more "aesthetic" now.

GJ! Cant wait to have a look.
